prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 
+# Non-zero exit code already if one command in a pipe fails
+set -o pipefail
```

Comment:
you can combine it to set -uo pipefail

Reply:
Actually, I changed it now so that one could disable the check just in case I have missed something and otherwise we would need a new O2 for the topology generation to work. After a few weeks we can add it all to one line